docs/src/modules/sdk/nav.adoc

```diff
@@ -35,5 +35,6 @@
 *** xref:sdk:auth-with-jwts.adoc[]
 *** xref:sdk:running-locally.adoc[]
 *** xref:sdk:model-provider-details.adoc[]
+** xref:sdk:sanitization.adoc[Data sanitization]
```

Comment:
be aware of that we also have it listed in docs/src/modules/sdk/pages/setup-and-configuration/index.adoc, but it's not in the nav of setup-and-configuration